Browser/generated/playwright_pb2_grpc.py

```diff
@@ -1,11 +1,12 @@
 # Generated by the gRPC Python protocol compiler plugin. DO NOT EDIT!
+"""Client and server classes corresponding to protobuf-defined services."""
```

Comment:
This would need to be added to the `playwright.proto` for it to be persistent across protobuf re-generations, otherwise it will just disappear. I don't actually know if the proto file supports adding a doc comment to the top of the generated files.

Reply:
Actually were these (and the changes in playwright_pb2.py) generated automatically from an updated version of the protobuf library? If so, It'd be good to also update requirements.txt with the newer grpc version.

Reply:
¯\_(ツ)_/¯